### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by yashodaAgarwal**
Please review my home, login and signup page

### CODE REVIEW

File: Styles/Login.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+.login-container {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  background-color: var(--white-color);
+  transform: translate(-50%, -50%);
+  width: 410px;
+  height: 450px;
+  padding: 30px;
+  box-shadow: 1px 1px 10px var(--black-color);
```
Comment (notmafiaa) [inline Styles/Login.css:24]: px values ? any specific reason you have used it? 

use rems instead. If possible!!

File: Styles/Login.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  font-weight: 800;
+}
+
+.login-container .login-heading {
+  font-size: 35px;
+}
```
Comment (notmafiaa) [inline Styles/Login.css:30]: check here too

File: Styles/Login.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+.login-container form .data {
+  height: 45px;
+  width: 100%;
+  margin: 40px 0;
+}
```
Comment (notmafiaa) [inline Styles/Login.css:36]: check here

File: Styles/Login.css [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+form .data input {
+  height: 100%;
+  width: 100%;
+}
+
+form .btn button {
+  margin-top: 15px;
+  padding: 10px;
```
Comment (notmafiaa) [inline Styles/Login.css:44]: check

File: Styles/Login.css [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+  color: var(--white-color);
+  border: none;
+}
+
+button:hover {
+  cursor: pointer;
+}
+.remember-me {
+  margin-right: 70px;
+}
```
Comment (notmafiaa) [inline Styles/Login.css]: check

File: Styles/Login.css [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+a {
+  color: var(--blue-color);
+  text-decoration: none;
+}
+
+.create-new-account {
+  text-align: center;
+  padding: 20px;
+}
```
Comment (notmafiaa) [inline Styles/Login.css:67]: check

File: Styles/home.css [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+} */
+.cake {
+  position: absolute;
+  width: 300px;
+  height: 200px;
+}
+.text {
+  position: relative;
+  top: 100px;
+  text-align: center;
```
Comment (notmafiaa) [inline Styles/home.css]: do check!!

File: Styles/home.css [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+.body-cart{
+  margin-bottom: 50px;
+  margin-top: 680px;
+  display: flex;
+  justify-content: space-around;
+}
+.body-cart-container{
+  padding: 5px;
+  display: flex;
+  width: 500px;
```
Comment (notmafiaa) [inline Styles/home.css]: do check the values!!

**Review by notmafiaa — CHANGES_REQUESTED** [review]
Please Check at the comments

File: Styles/home.css (hunk 8 above)
Comment (yashodaAgarwal) [reply Styles/home.css]: changed

**Comment by ArpitPatni** [comment]
Use rem or % in place of px (just a suggestion).Otherwise good to merge

**Pull request merged**